internal/iptables/iptables.go

```diff
@@ -270,12 +263,16 @@ func (fc FirewallConfiguration) makeIgnoreUserID(chainName string, uid int, comm
 		"--comment", formatComment(comment))
 }
 
-func (fc FirewallConfiguration) makeCreateNewChain(name string, comment string) *exec.Cmd {
+func (fc FirewallConfiguration) makeFlushChain(name string) *exec.Cmd {
 	return exec.Command(fc.BinPath,
 		"-t", "nat",
-		"-N", name,
-		"-m", "comment",
-		"--comment", formatComment(comment))
+		"-F", name)
+}
+
+func (fc FirewallConfiguration) makeCreateNewChain(name string) *exec.Cmd {
```

Comment:
Would you please add a comment flag here?

Reply:
I removed it because as far as I can see, although iptables accepts `-m comment --comment` with the `-N` command, it doesn't actually do anything with a comment specified on a chain.  For example, running `iptables-save` doesn't show the comment subsequently.  Also `man iptables-extensions` states "comment: Allows you to add comments ... to any **rule**" (emphasis mine).

iptables is really picky about matching the exact specification of a rule when deleting it - i.e. if you run `-A foo -m comment --comment bar`, you must run `-D foo -m comment --comment bar`, but this is not the case with chains.

Because of this difference in behavior, I thought it was clearer to omit the comment on `-N`, which is unhonored anyway, just as there is no logic to try to retrieve the comment previously used when running `-F`.

Please reconfirm if you want the comment parameter readded?